include/dt-bindings/interrupt/tegra186-irq.h

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
We don't use defines for IRQ numbers; IRQ numbers come directly from the HW, so are included in DT files as raw integers. We use defines for other things (clock/reset numbers for example) since those are numbered using an "artificial" SW scheme that doesn't correspond directly to HW.